crates/fluvio-protocol/src/link/error_code.rs

```diff
@@ -182,6 +182,14 @@ pub enum ErrorCode {
     #[fluvio(tag = 9000)]
     #[error("a compression error occurred in the SPU")]
     CompressionError,
+
+    // Deduplication
+    #[fluvio(tag = 10000)]
```

Comment:
We should avoid adding new error codes until a new strategy is decided.  Not show stopper

Reply:
Ok. Especially, on SPU/SC there is no much sense in enum of errors.